source/index.rst

```diff
@@ -30,5 +30,6 @@ Table of contents
 
    introduction
    concepts/state-channel
+   concepts/protocols
```

Comment:
At least in my browser, the TOC does not seem to show as intended. It is showing section "Protocols" with only one direct subsection, "Types of state channels". The protocol descriptions are currently not accessible.

Reply:
Fixed the toc tree.

**Problem:**
If types of channel and channel app are kept in the same page;
  and,
1. If the toc tree is placed before these headings, "types of
channels" and "channel app" appear after "protocols" in the toc 
tree.
2. If the toc tree is placed after these headings, "protocols"
appears as a sub-section of "channel app"

**Fix:**
In a toctree, only files can be included. So move "types of channels"
  and "channel app" to separate files and place them in a hidden toc 
  tree under the protocols section.

Reply:
Had similar problems before. This is something I don't like how it works. I feel it's much easier to structure your document with latex for example. Anyhow, now looks good.

Reply:
Yeah, I also find this aspect of sphinx restrictive :(.